### PR TITLE
fix: standardize default export option to "visited"

### PR DESCRIPTION
**Pull request by mydiemho**
AB#17532

### CODE REVIEW

File: src/providers/export/azureCustomVision.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
                 "visited",
                 "tagged"
             ],
-            "default": "all",
+            "default": "visited",
```
Comment (mydiemho) [inline src/providers/export/azureCustomVision.json]: this doesn't seem to make any difference

Comment (wbreza) [reply src/providers/export/azureCustomVision.json]: @mydiemho What do you mean?  This should make all the difference regarding how the form controls get initialized.

File: src/react/components/pages/export/exportForm.tsx [hunk 2]
```diff
@@ -160,8 +159,9 @@ export default class ExportForm extends React.Component<IExportFormProps, IExpor
 
         const formData = { ...exportFormat };
         if (resetProviderOptions) {
-            formData.providerOptions = { assetState: ExportAssetState.All };
+            formData.providerOptions = { assetState: ExportAssetState.Visited };
```
Comment (mydiemho) [inline src/react/components/pages/export/exportForm.tsx]: > @mydiemho What do you mean? This should make all the difference regarding how the form controls get initialized.

@wbreza  what I mean is that this is the code that set the `Provider Options` to the right default when I switch `Provider Type`.  The `default` in json file doesn't do anything

File: src/react/components/pages/export/exportForm.tsx [hunk 3]
```diff
@@ -160,8 +159,9 @@ export default class ExportForm extends React.Component<IExportFormProps, IExpor
 
         const formData = { ...exportFormat };
         if (resetProviderOptions) {
-            formData.providerOptions = { assetState: ExportAssetState.All };
+            formData.providerOptions = {} as IExportProviderOptions;
```
Comment (mydiemho) [inline src/react/components/pages/export/exportForm.tsx]: clear out previous settings and let the form component pull default from the export provider json file

Comment (wbreza) [reply src/react/components/pages/export/exportForm.tsx]: Does this work as expected to respect the default values?

Comment (mydiemho) [reply src/react/components/pages/export/exportForm.tsx]: > Does this work as expected to respect the default values?

yup it does :D

**Pull request merged**